proto/proto/responses.proto

```diff
@@ -113,3 +113,8 @@ message ListNotesResponse {
     // Lists all notes of the current chain
     repeated note.Note notes = 1;
 }
+
+message GetPublicAccountDetailsResponse {
+    // Public account full details.
```

Comment:
Please add a bit of documentation explaining what is the type of the encoded data. Just enough context for someone that will use this type to know how to decode it.

Reply:
This code was removed later.